webapp/src/index.tsx

```diff
@@ -103,6 +95,14 @@ root.render(
               element={<ComponentsBookPage />}
             />
           )}
+          <Route
+            path="*"
+            element={
+              <RequireAuth>
+                <Navigate to="/" />
+              </RequireAuth>
+            }
+          />
```

Comment:
If route does not exist, navigate to default

Reply:
question: What do we think about displaying a `NotFoundPage`?

Reply:
Nice idea! I've created a simple one :)